com.microsoft.teams.Teams.json

```diff
@@ -0,0 +1,106 @@
+{
+  "id": "com.microsoft.teams.Teams",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "19.08",
+  "sdk": "org.freedesktop.Sdk",
+  "base": "org.electronjs.Electron2.BaseApp",
+  "base-version": "19.08",
+  "command": "teams",
+  "tags": [
+    "proprietary"
+  ],
+  "separate-locales": false,
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--device=all",
+    "--filesystem=home",
+    "--talk-name=org.freedesktop.Notifications",
+    "--talk-name=org.kde.StatusNotifierWatcher",
+    "--talk-name=org.gtk.Notifications"
+  ],
+  "modules": [
+    {
+      "name": "ar",
+      "buildsystem": "simple",
+      "build-commands": [
```

Comment:
Can you just put it into build commands of Teams?

Reply:
I prefer to keep it separated. For me compiling or installing a dependency from the SDK is the same and should be kept in its own module.